wayfinder/manifest.json

```diff
@@ -0,0 +1,50 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "a68bad83-ba55-4350-a913-2f98bb667bad",
+  "app_id": "wayfinder",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Send Wayfinder metrics to Datadog",
+    "title": "wayfinder",
+    "media": [],
```

Comment:
Please add images that meet our media guidelines as described in the IMAGES_README file. We recommend adding screenshots of data flowing into your dashboard, or of your platform.

Reply:
Done